robber/matchers/callable.py

```diff
@@ -0,0 +1,18 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class Callable(Base):
+    """
+    expect(function).to.be.callable()
+    """
+
+    def matches(self):
+        return hasattr(self.actual, '__call__')
```

Comment:
I'm doing some quick googling and it looks like this is not all the cases. They are still checking for another condition to make sure it's callable. And I found out that we might be able to use `callable` builtin function. Wdyt?

My ref:
http://stackoverflow.com/questions/111234/what-is-a-callable-in-python
https://docs.python.org/2/library/functions.html#callable